css/index.css

```diff
@@ -25,6 +27,22 @@ a:hover {
   background-size: cover;
   box-sizing: border-box; }
 
+.hackclub-logo {
+  font-family: "Amaranth", sans-serif;
+}
```

Comment:
Is the webfont for Amaranth being loaded on the page? If not, this won't render correctly.